RedHatInsights/patchman-ui
# feat(SPM-1770): introduce smart management check for Templates page

# Description

Associated Jira ticket: # [SPM-1770](https://issues.redhat.com/browse/SPM-1770)

As a permissioned user who does not have a Smart Management entitlement, when I try to visit Patch templates, I should see that this is Smart Management functionality and be given an opportunity to do something about that, so that I can check it out and decide if I want to buy Smart Management entitlements to keep using this awesome feature. 

# How to test the PR
I will try to create an account without smart management entitlement for testing the PR. I will update once I can create.

- A user with proper permissions, but on an account WITHOUT a Smart Management entitlement, should be see an entitlement blocker page when visiting /insights/patch/templates.

- A user with proper permissions and on an account WITH a Smart Management entitlements, should be able to access /insights/patch/templates as normal.

# Before the change


# After the change


# Dependent work link


# Checklist:
